.github/CONTRIBUTING.md

```diff
@@ -55,19 +55,33 @@ git checkout -b branch-name
 
 - Create a new file for your article/blog in the `_posts` folder. The format of your file should be like `yyyy-mm-dd-title-of-article.md`, following this naming convention is very important as jekyll would not be able to read your post otherwise.
 
-- In your article/blog file, add the following jekyll-liquid header at the beginning of your file, which is required, without it jekyll would not be able to read your post.
+- If you're a frequent blogger, then add your profile in `_data/authors.yml` file and if you don't want to do that, add the following fields to the YAML frontmatter of the post.
```

Comment:
Hey @PratikshaJain37, this work has already been done in a separate PR, I think you've created new commits after fetching the upstream changes. Now these have become yours and are showing up here. This shouldn't be the case as your PR should only change files which are related to your feature.